orttraining/orttraining/test/python/orttraining_ortmodule_tests.py

```diff
@@ -150,6 +150,14 @@ def run_pytorch_export_contrib_ops_tests(cwd, log):
     run_subprocess(command, cwd=cwd, log=log).check_returncode()
 
 
+def run_lstm_training_op_tests(cwd, log):
+    log.debug("Running: LSTM Training Ops Tests")
+
+    command = [sys.executable, "-m", "pytest", "-sv", "orttraining_test_lstm.py"]
```

Comment:
we are adding two new operators, why we need a python test? any differences it tests compared with C++ UTs?

Reply:
The test adds reference implementation of the LSTM and LSTMGrad in numpy.
The idea in the test is to compare the outputs from the ort with the reference numpy implementation.
I can do this in python with ease since I can generate random data, run with ort, run with np, and compare. I cannot do this on C++ side.
If I add this on C++ side, I will have to add some form of "magic/golden numbers" in some file and always use that to run the test.

Any concern with adding the test in python?